stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -43,14 +43,17 @@ public data class Attachment(
 
     @IgnoreSerialisation
     @IgnoreDeserialisation
+    @Transient
```

Comment:
To make this class serializable by Moshi we need to mark as non-serializable some fields